sentry/src/routes/channel.rs

```diff
@@ -498,6 +504,68 @@ pub async fn channel_payout<C: Locked + 'static>(
     Ok(Json(SuccessResponse { success: true }))
 }
 
+/// GET `/v5/channel/0xXXX.../get-leaf/spender` request and
+/// GET `/v5/channel/0xXXX.../get-leaf/earner` request
```

Comment:
I would say we should do the same as in the route documentation - "GET `/v5/channel/0xXXX.../get-leaf`" and list the 2 routes for earner & spender below in a list.